### PR TITLE
Build branch

### PR DESCRIPTION
**Pull request by rloterh**
- Created 2 bubble sort methods: Bubble_sort and Bubble_sort by
- All test cases running successfully

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        temp = sample_array[j]
+        sample_array[j] = sample_array[j + 1]
+        sample_array[j + 1] = temp
+      end
+      j += 1
+    end
+    i += 1
+  end
+  puts sample_array
+end
```
Comment (euqueme) [inline bubble_sort.rb]: - [ ] bubble_sort_by is failing when given an array of Integers let's fix it 

image

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @rloterh :lollipop: 

### Thank you for your submission :handshake: you've done an amazing job with your `bubble_sort` and `bubble_sort_by` methods you've also had follow Gitflow correctly and your Readme file is professional :necktie: even including a live demo on repl.it so claps to you for that :clap: :clap: :clap: there's just a minor change to make to your `bubble_sort_by` method I've left a comment in your code bellow for you to address it :point_down: 

### That's it! :raised_hands: 

### Please :pray: do the changes :technologist:  and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Comment by rloterh** [comment]
- The code follows the example code given and works fine
- The second method, 'bubble_sort_by' was not to be tested on integers since the length property was used in the example

**Comment by rloterh** [comment]
-The second method, 'bubble_sort_by' follows the example given in the code
-The 'bubble_sort_by' method was to be tested against integer arrays since length property was used, as can be seen in the example given

**Review by mcrd25 — APPROVED** [review]
# Changes Approved :heavy_check_mark: 

Hi @rloterh :wave: 

Great job!
You were right about `bubble_sort_by` not being meant for integers, that was our bad! :sweat_smile: 
No need for further review! :+1: 

***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

**Comment by rloterh** [comment]
All criteria implemented

**Pull request closed without merging**